### DISCUSSION TITLE
Comments on the website

### DISCUSSION THREAD

**Discussion by LITUATUI** [LITUATUI/M3UPT#22] [General]
Comments made on the website will be stored here.

**Comment by FredRosa** [comment]
Obrigado por este excelente trabalho! Houve aqui uma grande sincronia entre tugas com o Jimmy Dore Show.

**Reply by LITUATUI** [reply]
Como comediante o Jimmy Dore faz melhor jornalismo que a maior parte dos jornalistas.

**Comment by frarodrigues1978** [comment]
Excelente trabalho. Alguém com uma lista do Brasil? Obrigado
